perf/security/file-mount-tests/non-sds-1/setup_test.sh

```diff
@@ -11,32 +11,18 @@ if [[ ! -d "${WD}" ]]; then
   mkdir $WD
 fi
 
-URL=""
-case "${OSTYPE}" in
-  darwin*)
-    if [[ "$RELEASETYPE" == "daily" ]]; then
-        URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
-    elif [[ "$RELEASETYPE" == "release" ]]; then
-        URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
-    elif [[ "$RELEASETYPE" == "pre-release" ]]; then
-        URL="https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
-    else
-        echo "Please specify RELEASETYPE"
-    fi ;;
-  linux*)
-    if [[ "$RELEASETYPE" == "daily" ]]; then
-        URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-    elif [[ "$RELEASETYPE" == "release" ]]; then
-        URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-    elif [[ "$RELEASETYPE" == "pre-release" ]]; then
-        URL="https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-    else
-        echo "Please specify RELEASETYPE"
-    fi ;;
-  *) echo "unsupported: ${OSTYPE}" ;;
-esac
-
-curl -JLo "$WD/istio-${RELEASE}.tar.gz" "${URL}"
+source ../../utils/get_release.sh
```

Comment:
Would it also make sense to move the remaining lines inside the common script and get rid of that redundancy as well.

Reply:
Done. Thanks.